src/Constants.ts

```diff
@@ -72,6 +69,9 @@ export const envVariableKeys = {
     THUNDRA_MASK_LAMBDA_PAYLOAD: 'thundra_agent_lambda_trace_integrations_aws_lambda_payload_mask',
     THUNDRA_MASK_HTTP_BODY: 'thundra_agent_lambda_trace_integrations_aws_http_body_mask',
     THUNDRA_MASK_ERROR_STACK_TRACE: 'thundra_agent_lambda_error_stacktrace_mask',
+
+    THUNDRA_AGENT_TIME_AWARE_SAMPLER_TIME_FREQ: 'thundra_agent_lambda_sample_sampler_timeAware_timeFreq',
+    THUNDRA_AGENT_COUNT_AWARE_SAMPLER_COUNT_FREQ: 'thundra_agent_lambda_sample_sampler_countAware_countFreq',
```

Comment:
To be consistent with Java and keep env. variable name shorter, `thundra_agent_lambda_sampler_countAware_countFreq` is better